internal/tools/aws/database.go

```diff
@@ -66,7 +68,27 @@ func (d *RDSDatabase) Teardown(keepData bool, logger log.FieldLogger) error {
 		return errors.Wrap(err, "unable to delete RDS DB cluster")
 	}
 
-	logger.Debug("AWS RDS DB cluster deleted")
+	logger.Debug("AWS RDS db cluster deleted")
+
+	resourceNames, err := d.getKMSResourceNames(awsID)
+	if err != nil {
+		return errors.Wrapf(err, "unabled to get KMS resources associated with db cluster %s", awsID)
+	}
+
+	if len(resourceNames) > 0 {
```

Comment:
The `else` in this statement can be eliminated by flipping this condition and returning early if `len(resourceNames) == 0`

Reply:
If I do that I will need to duplicate `logger.Debug("AWS RDS db cluster teardown completed")`. Also, I'd rather having only one returning point for teardown complete.